src/Provider/Doctrine/Persistence/Helper/PlatformHelper.php

```diff
@@ -104,11 +104,11 @@ public static function getOracleMysqlVersionNumber(string $versionString): strin
     public static function getMariaDbMysqlVersionNumber(string $versionString): string
     {
         preg_match(
-            '#^(?:5\.5\.5-)?(mariadb-)?(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)#i',
+            '#^(?:5\.5\.5-)?(mariadb-)?(?P<major>\d+)\.(?P<minor>\d+)(\.(?P<patch>\d+))?#i',
```

Comment:
Well, I assume you're right even if this regex comes from `doctrine/dbal` itself: https://github.com/doctrine/dbal/blob/7a8252418689feb860ea8dfeab66d64a56a64df8/src/Driver/AbstractMySQLDriver.php#L96

Reply:
I just got the error when I was running tests.
![CleanShot 2024-09-10 at 14 09 09@2x](https://github.com/user-attachments/assets/0427cfd9-562e-42da-ab36-01f64ffa33eb)


`$versionString`, as reported by a local postrgres running in a container, was "13.16"

That string matches what I'd expect when checking postgres releases:
https://www.postgresql.org/docs/release/

Maybe the problem is that this function was called to parse a postgres version, and it should be reserved for a MariaDb version.

Reply:
When I changed the regex I was able to run my project's tests again.

Reply:
Yeah, on further inspection, I think the problem is that this version-parsing is being called when not needed. I believe the problem may come from `isJsonSupported()`.

Reply:
You're right, `isJsonSupported()` seems to be the culprit, will post a fix soon.
Would you mind dumping `$version` after https://github.com/DamienHarper/auditor/blob/9e02fc6249c9157877c9f60bccb925f6c0e115b7/src/Provider/Doctrine/Persistence/Helper/PlatformHelper.php#L57

Reply:
Thanks, I hadn't understood you got the version that way ;)

Reply:
Will release it asap